OpenQA/Isotovideo/Interface.pm

```diff
@@ -22,7 +22,7 @@ use warnings;
 # -> increment on every change of such APIs
 # -> never move that variable to another place (when refactoring)
 #    because it may be accessed by the tests itself
-our $version = 20;
+our $version = 22;
```

Comment:
Why not just incrementing the version by one?

Reply:
That's done in two individual commits, one for the changed handling and optional command line parameter, one for the additional function